Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -11,15 +11,22 @@ import Zip
 enum FileLoggerError: Error {
     case missingWritableFileHandle
     case stringToDataConversionFailure
+    case userDefaultsInitFailure
+}
+
+public enum SharingConfiguration {
+    case shared(appGroupID: String)
+    case nonShared(suiteName: String)
+    case defaultNonShared
```

Comment:
`// Resources are not shared between the application and its extension. UserDefaults.standard is used.`

Reply:
pridano